staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1734,6 +1734,38 @@ type StartOptions struct {
 	DryRun []string `json:"dryRun,omitempty" protobuf:"bytes,5,rep,name=dryRun"`
 }
 
+// PauseOptions may be provided on pause request.
+//
+// +k8s:openapi-gen=true
+type PauseOptions struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// When present, indicates that modifications should not be
+	// persisted. An invalid or unrecognized dryRun directive will
+	// result in an error response and no further processing of the
+	// request. Valid values are:
+	// - All: all dry run stages will be processed
+	// +optional
+	// +listType=atomic
+	DryRun []string `json:"dryRun,omitempty" protobuf:"bytes,1,rep,name=dryRun"`
+}
+
+// UnpauseOptions may be provided on unpause request.
+//
+// +k8s:openapi-gen=true
```

Comment:
Is this necessary?

Reply:
Thanks. I think it is not necessary, probably a mistake too. I will fix it